pkg/virt-api/webhooks/validating-webhook/admitters/instancetype-admitter.go

```diff
@@ -53,16 +54,54 @@ func admitInstancetype(ar *admissionv1.AdmissionReview, expectedGvr metav1.Group
 		)
 	}
 
-	_, err := extractInstancetype(ar.Request.Object.Raw)
+	instancetypeObj, err := extractInstancetype(ar.Request.Object.Raw)
 	if err != nil {
 		return webhookutils.ToAdmissionResponseError(err)
 	}
 
+	causes := ValidateInstanceTypeSpec(k8sfield.NewPath("spec"), &instancetypeObj.Spec)
+	if len(causes) > 0 {
+		return webhookutils.ToAdmissionResponse(causes)
+	}
+
 	return &admissionv1.AdmissionResponse{
 		Allowed: true,
 	}
 }
 
+func ValidateInstanceTypeSpec(field *k8sfield.Path, spec *instancetypev1alpha2.VirtualMachineInstancetypeSpec) []metav1.StatusCause {
+	var causes []metav1.StatusCause
+
+	causes = append(causes, validateMemoryOvercommitPercentSetting(field, spec)...)
+	causes = append(causes, validateMemoryOvercommitPercentNoHugepages(field, spec)...)
+	return causes
+}
+
+func validateMemoryOvercommitPercentSetting(field *k8sfield.Path, spec *instancetypev1alpha2.VirtualMachineInstancetypeSpec) (causes []metav1.StatusCause) {
+	if spec.Memory.OvercommitPercent < 0 {
```

Comment:
How about adding an upper bound here?

`>=100` is obviously wrong. Maybe even `>90` should be disallowed / raise a warning